nodejs_mic_vad_streaming/Readme.md

```diff
@@ -5,40 +5,72 @@ DeepSpeech with voice activity detection.
 
 ### Prerequisites:
 
-1) The example utilized the [mic](https://github.com/ashishbajaj99/mic) NPM module which requires
-either [sox](http://sox.sourceforge.net/) (Windows/Mac) or [arecord](http://alsa-project.org/) (Linux).
+Node LTS v14.14 installed:
+```bash
+nvm install 14.14
```

Comment:
I would advise against documenting `nvm` out of hand like that. Either we give more steps / link to upstream doc, or we don't document with `nvm` at all.